docs/topics/mta-web-reviewing-apps.adoc

```diff
@@ -11,16 +11,23 @@ You can use the {ProductName} ({ProductShortName}) {WebName} to determine the mi
 .Procedure
 
 . In *Migration* view, click *Application inventory*.
-+
-// ![](/Tackle2/AppAssessAnalyze/AnalysisSelect.png)
 . Select the application you want to review.
 +
 [NOTE]
 ====
 Only one application can be reviewed at a time.
 ====
-. Click  *Review*.
-. Select *Proposed action* and *Effort estimate* from the lists. Set *Business criticality* and *Work priority*.
-. Click *Submit review*.
+
+. Click image:kebab.png[Options menu] at the right end of the row, and select *Review* from the drop-down menu. The application *Review* parameters appear in the main pane.
 +
+image::mta-web-assessment-review.png[Application review]
+
+. Click *Proposed action* and select the action.
+. Click *Effort estimate* and set the level of effort required to perform the assessment with the selected questionnaire.
+
```

Comment:
Hi @HagayVider1, looks like there is an redundant empty line here
Other than that, LGTM, thanks.